File: Modul-I/C#PartTwo/Homework/Methods/LargerThanNeighbours/LargerThanNeighbours.cs
namespace LargerThanNeighbours
{
    using System;
    using System.Linq;

    public class LargerThanNeighbours
    {
        public static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());

            int[] numbers = Console.ReadLine()
                .Split(' ')
                .Select(int.Parse)
                .ToArray();
            int largerThanNeighbourCount = LargerThanNeighboursCount(numbers);

            Console.WriteLine(largerThanNeighbourCount);
        }

        private static int LargerThanNeighboursCount(int[] numbers)
        {
            int largerThanNeighbpurCount = 0;
            for (int i = 0; i < numbers.Length; i++)
            {
                int currentNumber = numbers[i];

                if (i != 0
                    && i < numbers.Length - 1
                    && numbers[i - 1] < currentNumber
                    && currentNumber > numbers[i + 1])
                {
                    largerThanNeighbpurCount++;
                }
            }

            return largerThanNeighbpurCount;
        }
    }
}

File: Modul-I/03.JavaScript-Fundamentals/Homeworks/08. Functions/05. Larger than neighbours/LargerTHanNeighbours.js
function solve(args) {
    var  numbers = args[1].split(' ').map(Number),
        largerThanNeighbpurCount,
        i,
        currentNumber,
        length,
        prevNumber,
        nextNumber;


    length = numbers.length;
    largerThanNeighbpurCount = 0;

    for (i = 1; i < length - 1; i += 1) {
        currentNumber = numbers[i];
        prevNumber = numbers[i - 1];
        nextNumber = numbers[i + 1];
        if (currentNumber > prevNumber && currentNumber > nextNumber) {
            largerThanNeighbpurCount += 1;
        }
    }

    return largerThanNeighbpurCount;
}